kano/network.py

```diff
@@ -233,8 +233,11 @@ def getList(self, unsecure=False, first=False, debug=False):
         '''
 
         def sortNetworks(adict):
-            x, z = adict['quality'].split('/')
-            factor = int(x) / float(z)
+            if '/' in adict['quality']:
+                x, z = adict['quality'].split('/')
+                factor = int(x) / float(z)
+            else:
+                factor = int(adict['quality']) / 100
```

Comment:
I think that you need to either `float(adict['quality'])` or use `100.`, else the factor will almost certainly always be an integer and hence 0.